src/app/common/models/course.model.ts

```diff
@@ -0,0 +1,13 @@
+import { LocationInformation } from './location-information.model';
+import { ClassInformation } from './class-information.model';
+
+export interface Course {
+  courseName: string;
+  type: string;
+  day: string;
+  time: string;
+  location: LocationInformation;
+  weeks: string;
+  class: ClassInformation;
```

Comment:
`class` is a reserved keyword - must be changed once a new version of the backend has been released.